cirq/sim/clifford/clifford_simulator.py

```diff
@@ -47,8 +48,16 @@ def __init__(self):
         self.init = True
 
     @staticmethod
-    def get_supported_gates() -> List['cirq.Gate']:
-        return [cirq.X, cirq.Y, cirq.Z, cirq.H, cirq.S, cirq.CNOT, cirq.CZ]
+    def is_supported_operation(op: 'cirq.Operation') -> bool:
```

Comment:
This handling is being replaced with a has_stabilizer_effect protocol in #2760

Reply:
That PR have not been merged yet. As soon as it's merged, I'll update this PR to extend `_has_stabilizer_effect_` on all matrices and to use it to check if gate can be simulated by Clifford simulator.

By the way, I think there is more elgeant way of implementing this - instead of defining `_has_stabilizer_effect_` at every gate, we can do the following:
In has_stabilizer_effect_protocol.py check if gate has 2x2 uniary matrix (name it U).
If so, check that `U@X@U.conj().T` and `U@Z@U.conj.T` are in set `X, -X, Y, -Y, Z, -Z`.

This is also better than trying to match unitary with 24 matrices, as I am doing in this PR.